packages/web-components/fast-components/src/radio-group/radio-group.spec.md

```diff
@@ -0,0 +1,85 @@
+# Radio group
+
+## Overview
+
+As defined by the W3C:
+> A radio group is a set of checkable buttons, known as radio buttons, where no more than one of the buttons can be checked at a time. Some implementations may initialize the set with all buttons in the unchecked state in order to force the user to check one of the buttons before moving past a certain point in the workflow.
+
+### Use Cases
+
+Radio group allows the user to be presented with a list of all the options visible which can facilitate the comparison of choice.
+  
+### Features
+
+- **Selection:** A radio group can only support single select
+
+
+### Prior Art/Examples
+- [Material UI](https://material-ui.com/api/radio-group/)
+- [Lightning Design](https://www.lightningdesignsystem.com/components/radio-group/)
+- [Carbon design](https://www.carbondesignsystem.com/components/radio-button/code)
+- [Ant Design](https://ant.design/components/radio/)
+- [Atlassian](https://atlaskit.atlassian.com/packages/core/radio)
+- [Windows (UWP)](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/radio-button)
+
+---
+
+## Design
+
+### API
+
+*The key elements of the component's public API surface:*
+
+**Radio Group**
+*Component name:*
+- `fast-radio-group`
+
+*Attributes:*
+- `selected-value` - allows user to set the initial radio selected within the group independent of value.
+- `readonly`
+  - If there is a selected radio button in the group it should be submitted with the form but none of the radio buttons should be editable.
+- `disabled`
+  - All of the radio buttons should be disabled from user interaction and will not be submitted with the form data.
+
+*Slots:*
+- default slot for radio items
+
+*Events*
```

Comment:
hmmm actually - I'm not sure this element should raise a change-event. This is really just a semantic grouping

Reply:
wouldn't that mean if someone wanted to listen when the user selection changes for the group they have to subscribe to each of the internal radio button change events?

Reply:
no they can still *listen* on the group

```js
document.querySelector('fast-radio-group')
    .addEventListener(
        "change", () => console.log("something inside of me fired a change event")
    )
```

Reply:
removed the event from here and updated the for the label slot